themes/openy_themes/openy_rose/templates/block/block-branch-contacts-info.html.twig

```diff
@@ -0,0 +1,103 @@
+{#
+/**
+ * @file
+ * Default theme implementation to display branch contacts info.
+ *
+ * Available variables:
+ * - branch_title: will return branch title
+ * - openy_branch_selector: will return rendered branch selector code
+ * - phone: will return branch phone number
+ * - fax: will return branch fax number
+ * - email: will return branch email
+ * - address_title: will return formatted branch address string
+ * - directions_url: will return link to Google Maps linked to location address
+ * - branch_hours: will return rendered branch hours widget
+ * - directions_field_url: will return directiods field url
+ * - directions_field_title: will return directiods field title
+ *
+ * @ingroup themeable
+ */
+#}
+<div{{ attributes.addClass('branch-header') }}>
+  <section class="hidden-xs desktop">
+    <div class="container">
+      <address class="row branch-header-inner branch-info" data-branch-name="{{ node.label }}">
+        <div class="column col-sm-3 col-md-2 branch-header__item">
+          <h4>{{ 'WELCOME TO THE'|t }}</h4>
+          <p>{{ branch_title }}</p>
+          <p class="branch-header__directions"><a href="{{ directions_field_url }}" title="{{ directions_field_title }}">{{ directions_field_title }}</a></p>
+        </div>
+        <div class="column col-sm-3 address-wrapper branch-header__item">
+          <h4>{{ 'ADDRESS'|t }}</h4>
+          <a target="_blank" href="{{ directions_url }}" title="{{ address_title }}">{{ address_title }}</a>
+        </div>
+        <div class="column col-sm-3 col-md-2 branch-header__item">
+          <h4>{{ 'PHONE'|t }}</h4>
+          <a href="tel:{{ phone }}">{{ phone }}</a>
+          {% if fax %}
+          <br/><a href="tel:{{ fax }}">{{ fax }}</a> (fax)
+          {% endif %}
+          {% if email is not empty %}
+            <h4 class="branch-header__email-header">{{ 'EMAIL'|t }}</h4>
+            <a href="mailto:{{ email }}">{{ email }}</a>
+          {% endif %}
+        </div>
+        <div class="column col-sm-3 col-md-2 branch-header__item">
+          <h4>{{ 'TODAY\'S HOURS'|t }}</h4>
+          {{ branch_hours }}
+        </div>
+        <div class="column col-sm-3 col-md-3 save-link branch-header__item">
+          {# @see openy_branch_selector module #}
+          {{ openy_branch_selector }}
+        </div>
+      </address>
+    </div>
+  </section>
+
+  <blockquote class="hidden-sm hidden-md hidden-lg mobile">
```

Comment:
is there a way to leverage HTML structure from the desktop instead of duplicating it for mobile?